src/components/Tabledata.tsx/TableDetails.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Person } from '../../types/Person';
+
+interface Props {
+  person: Person;
+  addClassToRow: (event: string) => void;
+  clickedSlug: string;
+}
+
+export const TableDetails: React.FC<Props> = ({
+  person,
+  addClassToRow,
+  clickedSlug,
+}) => {
+  const {
+    born, died, sex, name, fatherName, motherName, slug,
+  } = person;
+
+  const handleClick = (event: string) => {
+    addClassToRow(event);
+  };
+
+  return (
+    <tr
+      data-cy="person"
+      className={clickedSlug === slug ? 'has-background-warning' : ''}
+    >
+      <td>
+        <Link
+          to={`/people/${slug}`}
+          onClick={() => handleClick(slug)}
```

Comment:
```suggestion
          onClick={handleClick(slug)}
```

Reply:
If i change onclick in your way, there is a problem with getting slug value.